pilot/pkg/config/kube/gateway/templates/service-export.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: multicluster.x-k8s.io/v1alpha1
+kind: ServiceExport
```

Comment:
Why is ServiceExport here ?

Reply:
I mean: we should not export it automatically AFAIK.

Reply:
Its not automatic, but allow users a way to export from the Gateway, vs needing to create a ServiceExport on their own (which somewhat leaks the fact there is a Service created).

Thinking about it more its probably not very useful